lighthouse-core/scripts/gcp-collection/fleet-status.sh

```diff
@@ -42,7 +42,7 @@ do
 
     if [[ "$EXTRA_ARG" == "--kill" ]]; then
       printf "Killing $instance...\n"
-      gcloud --project="$CLOUDSDK_CORE_PROJECT" compute instances delete "$instance" --zone="$ZONE"
+      echo y | gcloud --project="$CLOUDSDK_CORE_PROJECT" compute instances delete "$instance" --zone="$ZONE"
```

Comment:
does gcloud cli really not have an option for non-interactive use? 

if so, comment that the y is to confirm killing the instance? :)

Reply:
try `-q` https://stackoverflow.com/questions/35900502/google-cloud-platform-delete-or-stop-instances-without-prompt